### DISCUSSION TITLE
Proactive Monitoring

### DISCUSSION THREAD

**Discussion by schie** [react-native-community/discussions-and-proposals#393] [Ideas]
Hello!

Does anyone have suggestions or experience w/ Monitoring tools for React Native?

I'm looking for something to monitor / proactively alert me of thrown errors and such. 

I've never worked w/ those types of things, so I don't know where to start. 

Thanks!

**Comment by pavan4git** [comment]
noo

**Comment by liamjones** [comment]
@schie We use [Sentry](https://sentry.io/) and have been pretty happy with it. Sentry also allows you to run the server software yourself if that's important to you.

I've also heard good things about [Bugsnag](https://www.bugsnag.com/) but haven't used it myself.

Both of these have a free tier you can try it out with (and maybe even stick with, depending on your usage). 

We previously used Firebase Crashlytics (formerly Fabric's Crashlytics) but found it a lot less insightful/useful with RN since there wasn't a native understanding of RN stack traces, etc (don't know if this has improved in more recent years).

**Reply by schie** [reply]
Thanks! I'll look into it.

**Comment by schie** [comment]
Sentry is really awesome.
I've been using it for around a month now.

Thanks @liamjones  !

**Reply by schie** [reply]
their documentation is also amazing.

**Reply by liamjones** [reply]
Yeah, a well documented project is always nicer to work with! 😄
